apps/fishing-map/features/map/overlays/track-corrections/TrackCorrectionsOverlay.tsx

```diff
@@ -69,7 +69,7 @@ const TrackCorrectionOverlayIssue = ({ issue }: { issue: TrackCorrection }) => {
       onMouseEnter={handleMouseEnter}
       onMouseLeave={handleMouseLeave}
       tooltip={t('trackCorrection.mapTooltip', {
```

Comment:
Discussed with @satellitestudiodesign and we think is better to have something simpler like [vesselName] - [issueType]